packages/features/form-builder/Components.tsx

```diff
@@ -391,7 +391,7 @@ export const Components: Record<FieldType, Component> = {
         if (!value) {
           setValue({
             value: options[0]?.value,
-            optionValue: "",
+            optionValue: options[0]?.optionValue || "",
```

Comment:
I don't understand this and other change in this component.

`optionValue` can only be updated through `ComponentForField` at line 469.

Reply:
The booking view expects the `optionValue` to have the a valid value. 
https://github.com/calcom/cal.com/blob/2ad3592eb39bbf8807e2bed9c7d78ae866cf1774/apps/web/modules/bookings/views/bookings-single-view.tsx#L132

Currently the `optionValue` is always empty regardless of the option selected.